src/google_form_to_engagement_db/google_form_to_engagement_db.py

```diff
@@ -170,6 +183,54 @@ def _form_answer_to_engagement_db_message(form_answer, form_id, form_response, p
     )
 
 
+def _all_equal(iterable):
+    """Checks if all elements in an iterable are identical"""
+    iterator = iter(iterable)
+    try:
+        first_item = next(iterator)
+    except StopIteration:
+        return True
+    return all(first_item == x for x in iterator)
+
```

Comment:
It might be worth just converting this to a list so we don't have to think about iterators:
iterable = list(iterable)
return all(x == iterable[0] for x in iterable)

Reply:
Updated, thanks.